packages/components/src/h-stack/stories/index.story.tsx

```diff
@@ -42,7 +42,8 @@ const JUSTIFICATIONS = {
 
 const meta: Meta< typeof HStack > = {
 	component: HStack,
-	title: 'Components (Experimental)/HStack',
+	title: 'Components (Experimental)/Layout/HStack',
+	id: 'components-hstack',
```

Comment:
As this component is still formally experimental, if we change its ID, it will cause the previous URL not to work:

http://localhost:50240/?path=/docs/components-experimental-hstack--docs

To fix that, we have 2 options:
- Use `id: 'components-experimental-hstack'`
- Add the component to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`. 

Since the component is still experimental, I'd prefer the former option. Once we stabilize the component, we can always add it to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`